src/components/left-nav.jsx

```diff
@@ -2,22 +2,32 @@ import React from 'react';
 import { NavLink } from 'react-router-dom'
 import Routes from './routes';
 
+import '../styles/left-nav.css';
+
+const leftNavPaddingClasses = 'ds-u-padding-right--3 ds-u-padding-left--3 ds-u-padding-top--1';
+
 const createLinksListItems = function(pathsMap) {
+  const linkClasses = 'ds-c-vertical-nav__label ' + leftNavPaddingClasses;
+
   return pathsMap.paths.map((path) => {
+      var linkComponent;
+      if (!path.external) {
+        linkComponent = <NavLink to={path.path} exact={path.exact} className={linkClasses} activeClassName='ds-c-vertical-nav__label--current'>{path.linkText}</NavLink>;
+      } else {
+        linkComponent = <a href={path.path} className={linkClasses}>{path.linkText}</a>;
+      }
       // <NavLink> is special version of the <Link> that will add styling
       // attributes to the rendered element when it matches the current URL
       // by applying activeClassName.
-      return <li key={path.path}>
-        <NavLink to={"/" + path.path} activeClassName=''>{path.linkText}</NavLink>
-      </li>
+      return <li className='ds-c-vertical-nav__item' key={path.path}>{linkComponent}</li>
   });
 }
 
 class NavSection extends React.Component {
   render() {
-    return <li className="ds-c-vertical-nav__item">
-      <ul className="ds-c-vertical-nav__subnav">
-        <li><b>{this.props.name}</b></li>
+    return <li>
+      <ul className="ds-c-vertical-nav">
```

Comment:
if we add a `left-nav` class here, we could avoid altering design system classes for left-nav specific styles (as described below)